src/ExtranetBundle/LoginByTransferToken/RouteGenerator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace ChameleonSystem\ExtranetBundle\LoginByTransferToken;
+
+use esono\pkgCmsRouting\CollectionGeneratorInterface;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+class RouteGenerator implements CollectionGeneratorInterface
+{
+    public const ROUTE_NAME = 'chameleon_system_extranet.login_by_transfer_token';
+    protected const CONTROLLER_SERVICE = 'chameleon_system_extranet.login_by_transfer_token.login_controller';
```

Comment:
No protected needed (?)

Reply:
:heavy_check_mark: Using `private` in place of `protected`. Old habits coming through here.